lindera-core/src/unknown_dictionary.rs

```diff
@@ -96,7 +96,12 @@ fn make_costs_array(entries: &[UnknownDictionaryEntry]) -> Vec<WordEntry> {
     entries
         .iter()
         .map(|e| {
-            assert_eq!(e.left_id, e.right_id);
+            // Do not perform strict checks on left context id and right context id in unk.def.
+            // Just output a warning.
+            //assert_eq!(e.left_id, e.right_id);
+            if e.left_id != e.right_id {
+                println!("Warning: left id and right id are not same: {:?}", e);
```

Comment:
You may use the crates [log](https://lib.rs/crates/log) allowing you to choose at which log level you want to print your message.

Combined with [env_logger](https://lib.rs/crates/env_logger), this allows users to choose which kind of messages are displayed.

Reply:
@ManyTheFish 
Thanks. I'll fix it.